tools/integration_tests/local_file/local_file_test.go

```diff
@@ -102,6 +102,11 @@ func TestMain(m *testing.M) {
 		{"--implicit-dirs=true", "--rename-dir-limit=3"},
 		{"--implicit-dirs=false", "--rename-dir-limit=3"}}
 
+	if !testing.Short() {
+		flags = append(flags, []string{"--implicit-dirs=false", "--rename-dir-limit=3", "--client-protocol=grpc"})
```

Comment:
Nit: you might be able to use https://github.com/GoogleCloudPlatform/gcsfuse/blob/bdcf1591778f7bb0a82963082391d35e67b2f570/tools/integration_tests/read_cache/setup_test.go#L121 (needs to be exported though)